staging/src/k8s.io/csi-translation-lib/plugins/aws_ebs_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+import "testing"
+
+func TestKubernetesVolumeIDToEBSVolumeID(t *testing.T) {
```

Comment:
could we add some negative test cases? e.g. malformed vol ID, some url that doesnt start with `aws` etc.